BlazarBase/src/main/java/com/hubspot/blazar/base/GitInfo.java

```diff
@@ -1,14 +1,15 @@
 package com.hubspot.blazar.base;
 
+import java.util.List;
+import java.util.Objects;
+
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.MoreObjects;
 import com.google.common.base.Optional;
 import com.google.common.base.Splitter;
 
-import java.util.List;
-import java.util.Objects;
-
 public class GitInfo {
```

Comment:
Does this object encapsulate the information for a Git Repository Branch? If yes it could be renamed to GitBranchInfo or GitBranch